lib/compressor/restructure/4-restructShorthand.js

```diff
@@ -121,6 +121,12 @@ TRBL.prototype.getValueSequence = function(value, count) {
             case 'Space':
                 return false; // ignore space
 
+            case 'Function':
+                switch (child.name) {
+                    case 'calc':
+                        special = child.name;
+                        break;
+                }
```

Comment:
```js
            case 'Number':
            case 'Percentage':
                break;   // <- should be blank line before `case`, look on other switch branches
            case 'Function':
                switch (child.name) {
                    case 'calc':
                        special = child.name;
                        break;
                }
                // <- here is should be break
            case 'Space':
                return false; // ignore space
```

Some things aren't covered by style checks. Look on other code is formatted in this case.